timescaledb/timescaledb-edition-comparison.md

```diff
@@ -5,39 +5,65 @@ keywords: [Apache, community, features]
-This is a classic open source license, meaning that it is completely unrestricted -
-anyone can take this code and offer it as-a-service.
+
+*   TimescaleDB with an Apache 2 licence
+*   TimescaleDB Community Edition
+
+The TimescaleDB Apache 2 Edition is the version of TimescaleDB that is available
+under the [Apache 2.0 license][apache-license]. This is a classic open source license,
+meaning that it is completely unrestricted - anyone can take this code and offer it
+as a service.
 
 ### TimescaleDB Apache 2 Edition
 
-- _Can I install TimescaleDB Apache 2 Edition in my own on-premises or cloud infrastructure and run it for free?_ <br/>
-  Yes.
-- _Can I sell TimescaleDB Apache 2 Edition as a service, even if I'm not the main contributor?_ <br/>
-  Yes.
-- _Can I modify the TimescaleDB Apache 2 Edition source code and run it for production use?_ <br/>
-  Yes.
+You can install TimescaleDB Apache 2 Edition on your own on-premises or cloud
+infrastructure and run it for free.
+
+You can sell TimescaleDB Apache 2 Edition as a service, even if you're not the
+main contributor.
+
+You can modify the TimescaleDB Apache 2 Edition source code and run it for
+production use.
+
+TimescaleDB Apache 2 Edition is available from these service providers:
 
-TimescaleDB Apache 2 Edition is available at the following service providers: [Azure Database for PostgreSQL][azure-database], [Digital Ocean][digital-ocean], [Aiven for PostgreSQL][aiven].
+*   [Azure Database for PostgreSQL][azure-database]
+*   [Digital Ocean][digital-ocean]
+*   [Aiven for PostgreSQL][aiven]
 
 ### TimescaleDB Community Edition
 
-TimescaleDB Community Edition is the latest, most updated version of TimescaleDB, available under the [Timescale License (TSL)][timescale-license]. [Read more about the Timescale License][license-blog].
+TimescaleDB Community Edition is the latest, most updated version of TimescaleDB,
+available under the
+[Timescale License (TSL)][timescale-license].
+
+For more information about the Timecale licence, see [this blog post][license-blog].
+
+Many of the most recent features of TimescaleDB are only available in
+TimescaleDB Community Edition.
+
+You can install TimescaleDB Community Edition in my own on-premises or cloud
+infrastructure and run it for free. TimescaleDB Community Edition is completely
+free if you manage your own service.
+
+You cannot sell TimescaleDB Community Edition as a service, even if you are the
+main contributor.
 
-Many of the most recent features of TimescaleDB are only available in TimescaleDB Community Edition.
+You can modify the TimescaleDB Community Edition source code and run it for
+production use. Developers using TimescaleDB Community Edition have the "right
+to repair" and make modifications to the source code and run it in their own
+on-premises or cloud infrastructure. However, consistent with the previous
+question, users may not make modifications to the TimescaleDB Community Edition
```

Comment:
There aren't any questions anymore so this is a bit confusing ;)

Reply:
oof, missed that bit 🤦‍♀️